packages/front-end/pages/features/[fid].tsx

```diff
@@ -41,6 +43,11 @@ export default function FeaturePage() {
     experiments: { [key: string]: ExperimentInterfaceStringDates };
   }>(`/feature/${fid}`);
 
+  const { features } = useFeaturesList();
+
+  const firstFeature = features?.length === 0;
```

Comment:
This is not a reliable way to do this. It's a race condition that will only work some of the time and worse may open the modal randomly on page load even if the person has many features already. Better to have the "create feature" modal add a query string when redirecting if it's the first feature.